stories/index.js

```diff
@@ -1,5 +1,65 @@
 // @flow
+import { storiesOf } from '@storybook/react'
+import { withKnobs } from '@storybook/addon-knobs'
+import { utils } from '../src'
 import '../src/index.css'
-import './grid'
-import './components'
-import './layout'
+
+/**
+ * Reads the `/components`, `/grid` and `/layout` subfolder files and adds them to their respective
+ * stories
+ */
+
+const folders = {
```

Comment:
could we do a `require.context` on a root folder that will parse all sub directories similar to what we are doing here? That way, we won't have to update this list of folders if we add additional story types.